examples/dask/start-with-dask.ipynb

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
<p>This isn't exactly accurate. Using <code>delayed</code>  doesn't by itself imply that something will be parallelized. I might write this like this:</p><p> </p><blockquote>You can use the <code>@dask.delayed</code>  decorator to change a regular Python function into a lazily-evaluated function. That means that a function call will return a future, instead of a value.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>